### ISSUE TITLE
[BUG] Pressing a specific key will make the list of stack names disappear in interactive mode. 

### ISSUE THREAD

**Issue by nao1215** [go-to-k/delstack#306]
## About the problem
The interactive mode of the delstack command displays a list of stack names. Below is an example:
```shell
? Select StackName.
Nested child stacks, XXX_IN_PROGRESS(e.g. ROLLBACK_IN_PROGRESS) status stacks and EnableTerminationProtection stacks are not displayed.
  [Use arrows to move, type to filter]
  stack-a
  stack-b
  stack-c
> stack-d
```

User select one stack from the above list.
When the user is choosing a stack name, pressing the [Enter] key is expected. However, if the user accidentally presses the **[Space] key**, the delstack display no stack name.  Below is an example:
```
? Select StackName.
Nested child stacks, XXX_IN_PROGRESS(e.g. ROLLBACK_IN_PROGRESS) status stacks and EnableTerminationProtection stacks are not displayed.
    [Use arrows to move, type to filter]


```

## Current workaround
If the user accidentally presses the [Space] key, pressing the [BackSpace] key will restore the display.

## Expected behavior
When a user presses an irrelevant key, the screen should indicate that the user pressed the wrong key or be unresponsive. At the very least, I want the stack name to continue to be displayed on the screen.

## Environment information
- macOS 13.6.3 (arm64)
- iTerm2 (Build 3.4.22)
- delstack version v1.4.1

**Comment by go-to-k** [comment]
@nao1215 

Thanks for the reporting! Glad you're using this tool.

I will take a look at this later, perhaps this feature could be implemented.

**Comment by go-to-k** [comment]
@nao1215 

I have tried to remove spaces from the beginning and end of the filtering word. Spaces in the middle are not removed, but this would not be a problem in the case of this issue. In addition, characters other than spaces are not covered, because it would be out of the concept of filtering.

We have released [v1.4.2](https://github.com/go-to-k/delstack/releases/tag/v1.4.2) which includes this fix.

If you see any problems, please let me know again. Thanks.

**Comment by nao1215** [comment]
@go-to-k 
Wow, thank you for the quick fix.
I have confirmed that there are no issues using the latest version. 

I will continue to use the delstack command from now on. This is very useful one.
